oneanddone/tasks/models.py

```diff
@@ -37,6 +37,7 @@ class Feedback(CachedModel, CreatedModifiedModel):
     text = models.TextField()
     time_spent_in_minutes = models.IntegerField(
         validators=[MinValueValidator(0), MaxValueValidator(99999)], blank=True, null=True)
+    rating = models.IntegerField(default = 0)
```

Comment:
+1 to this. Good idea to make the default `0` so that any that haven't been rated won't affect the total.